### PR TITLE
Add FTPToS3Operator

### PR DESCRIPTION
**Pull request by JavierLopezT**
New transfer

### CODE REVIEW

File: tests/providers/amazon/aws/transfers/test_ftp_to_s3.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import unittest
+from unittest import mock
+
+from airflow.providers.amazon.aws.transfers.ftp_to_s3 import FTPToS3Operator
+
+TASK_ID = 'test_s3_to_ftp'
```
Comment (subkanthi) [inline tests/providers/amazon/aws/transfers/test_ftp_to_s3.py]: task id should probably be ftp to s3.

**Review by subkanthi — CHANGES_REQUESTED** [review]

File: airflow/providers/amazon/aws/transfers/ftp_to_s3.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    :type ftp_conn_id: str
+    :param ftp_path: The ftp remote path, including the file.
+    :type ftp_path: str
+    :param aws_conn_id: The s3 connection id. The name or identifier for
+        establishing a connection to S3
+    :type aws_conn_id: str
+    :param s3_bucket: The targeted s3 bucket in which upload the file to
+    :type s3_bucket: str
+    :param s3_key: The targeted s3 key. This is the specified file path for
+        uploading the file from S3.
```
Comment (dstandish) [inline airflow/providers/amazon/aws/transfers/ftp_to_s3.py]: file _to_ s3

**Review by dstandish — CHANGES_REQUESTED** [review]

**Comment by JavierLopezT** [comment]
All checks have passed. Could anyone approve this, please? Thanks

**Comment by JavierLopezT** [comment]
> All checks have passed. Could anyone approve this, please? Thanks

Anyone? Thanks

File: airflow/providers/amazon/aws/transfers/ftp_to_s3.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+from airflow.providers.amazon.aws.hooks.s3 import S3Hook
+from airflow.providers.ftp.hooks.ftp import FTPHook
+from airflow.utils.decorators import apply_defaults
+
+
+class FTPToS3Operator(BaseOperator):
+    """
+    This operator enables the transferring of files from FTP server to S3.
+
+    :param ftp_conn_id: The ftp connection id. The name or identifier for
```
Comment (dstandish) [inline airflow/providers/amazon/aws/transfers/ftp_to_s3.py:37]: i think the params in doc should have same order as in `__init__`

**Review by dstandish — CHANGES_REQUESTED** [review]
LGTM aside from doctring param ordering

**Review by dstandish — APPROVED** [review]

**Pull request merged**